plugins/cros_ec-usb/fu-cros_ec-usb-common.c

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2020 Benson Leung <bleung@chromium.org>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <string.h>
+
+#include "fu-cros_ec-usb-common.h"
+
+gboolean
+fu_cros_ec_usb_parse_version (const gchar *version_raw,
```

Comment:
I think it would be reasonable to pass a `GError` as a 3rd argument for this function.  Then in all the `FALSE` conditions you can add more detail about what in the parsing actually failed.

Reply:
Done.